### ISSUE TITLE
[BUG] System.ComponentModel.Win32Exception: Acesso negado.

### ISSUE THREAD

**Issue by casengabr** [bloxstraplabs/bloxstrap#4135]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [X] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.3

### What problem did you encounter?

when i play some map in roblox, the launcher show me this mensage "System.ComponentModel.Win32Exception: Acesso negado."

### Bloxstrap Log

```text
2024-12-27T19:01:23Z [App::OnStartup] Starting Bloxstrap v2.8.3
2024-12-27T19:01:23Z [App::OnStartup] Compiled Friday, 27 December 2024 at 3:10:17 PM from commit a7e28200a9b14b94474dade8f0e7bb192e413067 (tag/v2.8.3)
2024-12-27T19:01:23Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-12-27T19:01:23Z [App::OnStartup] Temp path is %UserProfile%\AppData\Local\Temp\Bloxstrap
2024-12-27T19:01:23Z [App::OnStartup] WindowsStartMenu path is %UserProfile%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
2024-12-27T19:01:23Z [LaunchSettings] Identifier 'player' is active with data
2024-12-27T19:01:23Z [LaunchSettings] Invalid argument: roblox-player:1+launchmode:play+gameinfo:s2ss2dRid7AxsJrOp-UnQhaHlxY8_VGOVkA0eMs86LmGgtT1UeGRSeoLfS1XIkXwgDuzCKHcs5UpdMgMWDg4ZSZWAfwfo5uTBbtILcx0qCCQbdBeW2GPUFsWr2rS6FIX_0ZSABy7PDLWYxXgwknhERzYHQaaTaYjfN2wkd4ip_vbI9a3HAB6DC1j-z7j2EVfleRe3R0bR0OTUFYLAwlzgmvGnI-ydckG27bpCvtfq1A+launchtime:1735326081477+placelauncherurl:https%3A%2F%2Fwww.roblox.com%2Fpt%2FGame%2FPlaceLauncher.ashx%3Frequest%3DRequestGame%26browserTrackerId%3D1721331644258012%26placeId%3D10450270085%26isPlayTogetherGame%3Dfalse%26referredByPlayerId%3D0%26joinAttemptId%3D3a61135c-0b90-4dc9-87fa-52433f740a3a%26joinAttemptOrigin%3DPlayButton+browsertrackerid:1721331644258012+robloxLocale:pt_br+gameLocale:pt_br+channel:+LaunchExp:InApp
2024-12-27T19:01:23Z [LaunchSettings::ParsePlayer] Got Roblox launch arguments
2024-12-27T19:01:23Z [App::OnStartup] Entering main logic
2024-12-27T19:01:23Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20241227T190123Z.log
2024-12-27T19:01:23Z [Logger::Initialize] Finished initializing!
2024-12-27T19:01:23Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-12-27T19:01:23Z [JsonManager<Settings>::Load] Loaded successfully!
2024-12-27T19:01:23Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-12-27T19:01:23Z [JsonManager<State>::Load] Loaded successfully!
2024-12-27T19:01:23Z [FastFlagManager::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-12-27T19:01:23Z [FastFlagManager::Load] Loaded successfully!
2024-12-27T19:01:23Z [FastFlagManager::SetValue] Setting of 'FLogNetwork' to '7' is pending
2024-12-27T19:01:23Z [FastFlagManager::SetValue] Setting of 'FFlagHandleAltEnterFullscreenManually' to 'False' is pending
2024-12-27T19:01:23Z [LaunchHandler::ProcessLaunchArgs] Opening bootstrapper (Player)
2024-12-27T19:01:23Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper
2024-12-27T19:01:23Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper dialog
2024-12-27T19:01:23Z [Bootstrapper::Run] Running bootstrapper
2024-12-27T19:01:23Z [Bootstrapper::SetStatus] Conectando-se ao Roblox...
2024-12-27T19:01:23Z [Deployment::TestConnection<https://setup.rbxcdn.com>] Connecting...
2024-12-27T19:01:23Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/versionStudio
2024-12-27T19:01:23Z [Deployment::InitializeConnectivity] Testing connectivity...
2024-12-27T19:01:23Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/versionStudio
2024-12-27T19:01:23Z [Deployment::InitializeConnectivity] Got https://setup.rbxcdn.com as the optimal base URL
2024-12-27T19:01:23Z [Bootstrapper::Run] Connectivity check finished
2024-12-27T19:01:23Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-12-27T19:01:23Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-12-27T19:01:23Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-12-27T19:01:23Z [Bootstrapper::CheckForUpdates] No updates found
2024-12-27T19:01:23Z [Bootstrapper::GetLatestVersionInfo] Got channel as production
2024-12-27T19:01:23Z [Deployment::GetInfo] Getting deploy info for channel production
2024-12-27T19:01:23Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-12-27T19:01:24Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-12-27T19:01:24Z [RegistryKeyEx::SetValueSafe] Writing '' to HKEY_CURRENT_USER\SOFTWARE\ROBLOX Corporation\Environments\RobloxPlayer\Channel\www.roblox.com
2024-12-27T19:01:24Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/version-b71c150c7c1f40de-rbxPkgManifest.txt
2024-12-27T19:01:24Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/version-b71c150c7c1f40de-rbxPkgManifest.txt
2024-12-27T19:01:24Z [Bootstrapper::SetStatus] Instalando {product}...
2024-12-27T19:01:24Z [LaunchHandler::LaunchRoblox] Bootstrapper task has finished
2024-12-27T19:01:24Z [LaunchHandler::LaunchRoblox] An exception occurred when running the bootstrapper
2024-12-27T19:01:24Z [App::FinalizeExceptionHandling] (0x80004005) System.ComponentModel.Win32Exception (5): Acesso negado.
   at System.Diagnostics.ProcessManager.OpenProcess(Int32 processId, Int32 access, Boolean throwIfExited)
   at System.Diagnostics.NtProcessManager.GetModules(Int32 processId, Boolean firstModuleOnly)
   at System.Diagnostics.NtProcessManager.GetFirstModule(Int32 processId)
   at Bloxstrap.Bootstrapper.KillRunningRobloxInDirectory(String path)
   at Bloxstrap.Bootstrapper.UpgradeRoblox()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.Bootstrapper.Run()
2024-12-27T19:01:24Z [HttpClientLoggingHandler::ProcessRequest] POST https://bloxstraplabs.com/metrics/post-exception
```

**Comment by nabheyka880** [comment]
if you have an antivirus, turn it off and then install bloxstrap

**Comment by bluepilledgreat** [comment]
Fixed in v2.8.4
